### ISSUE TITLE
Long operation: getSemanticTokens 

### ISSUE THREAD

**Issue by itsToggle** [microsoft/pylance-release#2391]
## Environment data

-   Language Server version: Pylance language server 2022.2.1
-   OS and version: Windows 10 
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.10.0 64-bit

## Expected behaviour

Im writing a script with sympy, and I want the variables to be colored correctly.

## Actual behaviour

the coloring of the variables takes ages (325158ms to be exact)

## Logs

```
[Info  - 16:10:05] (11716) Pylance language server 2022.2.1 (pyright 618f54ed) starting
[Info  - 16:10:05] (11716) Server root directory: c:\Users\Ronald\.vscode\extensions\ms-python.vscode-pylance-2022.2.1\dist
[Info  - 16:10:05] (11716) No configuration file found.
[Info  - 16:10:05] (11716) No pyproject.toml file found.
[Info  - 16:10:05] (11716) Setting pythonPath for service "Desktop": "C:\Users\Ronald\AppData\Local\Programs\Python\Python310\python.exe"
[Warn  - 16:10:05] (11716) stubPath c:\Users\Ronald\Desktop\typings is not a valid directory.
[Info  - 16:10:05] (11716) Assuming Python version 3.10
[Info  - 16:10:05] (11716) Assuming Python platform Windows
[Info  - 16:10:05] (11716) Searching for source files
[Info  - 16:10:05] (11716) Found 9 source files
[Info  - 16:10:05] (11716) Background analysis(1) root directory: c:\Users\Ronald\.vscode\extensions\ms-python.vscode-pylance-2022.2.1\dist
[Info  - 16:10:05] (11716) Background analysis(1) started
[Info  - 16:10:06] (11716) Indexer background runner(2) root directory: c:\Users\Ronald\.vscode\extensions\ms-python.vscode-pylance-2022.2.1\dist (index)
[Info  - 16:10:06] (11716) Indexing(2) started
[Info  - 16:10:07] (11716) Indexer done(2). indexed 448 files
[Info  - 16:15:31] (11716) [BG(1)] Long operation: getSemanticTokens full at c:\Users\Ronald\Desktop\simplifiy_Gk_Gg.py (325158ms)
```

## Code Snippet / Additional information

```python
from sympy import *
from sympy.parsing.sympy_parser import parse_expr
import copy
```

**Comment by bschnurr** [comment]
can't repo. most likely fixed in next version today.

**Comment by debonte** [comment]
This issue has been fixed in version 2022.2.3, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202223-16-February-2022)
